Refine results

jturtle opened the pull request:
Hi Lucie,
Thanks for the explanation about parsing the constant-trajectory warnings.  While looking into this, I found that we could improve our order of operations.  This new submission is qualitatively the same, but reduces the number of numeric artifacts.  There are still a hand full of constant trajectories for each of DC and WY, 0-64 mortality, but I have verified that these are non-constant and reasonable before rounding.

Thanks,
Jamie

LucieContamin commented:
Hi @scc-usc ,

Thank you for your pull-request.
As I understand, the file with the suffix `_Dec` contains data for the scenario from December and the file with the suffix `_Jan` contains data for the scenario from January. However, the `_Jan` file contains the `scenario_name` for January but the column `scenario_id` contains identifier for the scenario from December and January. Would it be possible to please verify it?

Thanks,
Lucie

scc-usc commented:
Hi Lucie,

Thanks for pointing it out. There was an error in naming the scenario ids for the US country-level in _Jan.csv file. I have fixed it. 
Now, _Dec file should only contain scenarios declared in Dec, and _Jan file should only contain scenarios declared in January.

Thanks,
Ajitesh

LucieContamin commented:
Hi Ajitesh,

Thank you for the update. 
I understand that the combined file is too large to be submitted. So, to inform you, we will accept the combined file in the `zip` format. For this pull-request I can merge it and merge the two files together in a `zip` format. 

Please feel free to let me know if you have any questions,
Many thanks,

Lucie

scc-usc commented:
Thanks, Lucie. Next time, I will submit a zipped combined file.

Ajitesh

LucieContamin commented:
No submission file in the standard  SMH file format found in the Pull-Request.

LucieContamin commented:
No submission file in the standard  SMH file format found in the Pull-Request.

LucieContamin commented:
No submission file in the standard  SMH file format found in the Pull-Request.

jturtle committed:
Refine results

We were rounding to 0.1 too early in the calculations.  This version reduces numeric rounding artifacts and the number trajectories that are constant when rounded.

LucieContamin commented:
Hi @jturtle ,

Thank you very much for the update and the information! It seems to be all good, I will merge it!

Best, Lucie
